static/css/add_tip.css

```diff
@@ -0,0 +1,5 @@
+#add_tip {
+    align-content: center;
+    margin-top: 5%;
+    margin-bottom: auto;
+}
```

Comment:
missing line

Reply:
In css files the formating is without a new line I think